plonetheme/onegov/patterns/settings.py

```diff
@@ -0,0 +1,42 @@
+from Products.CMFPlone.interfaces import ILinkSchema
+from Products.CMFPlone.interfaces import IPatternsSettings
+from plone.registry.interfaces import IRegistry
+from zope.component import getUtility
+from zope.interface import implementer
+import json
+
+
+@implementer(IPatternsSettings)
+class OneGovPatternSettingsAdapter(object):
```

Comment:
I don't get, why we can't just subclass the referenced class in CMFPlone (patterns.settings.PatternSettingsAdapter).
Because except for the call method the exact same stuff is done. Or is it important not to have the other methods in here?

Maybe we could look at it together so I can understand the issue better.

Reply:
Good qu, but I'm with @Nachtalb here.  You only save one method - and you have an extra lookup to do to work out what's going on